examples/yew-tailwindcss/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "yew-example"
+version = "0.1.0"
+authors = ["Anthony Dodd <dodd.anthonyjosiah@gmail.com>"]
```

Comment:
Probably need to update the name here to `yew-tailwindcss`, and feel free to use your name as the author `:)`.

Reply:
I'm updating the name to be in line with the other examples (ie. `yew-tailwindcss-example`). I'm just wondering if it's worth removing the authors field altogether as that's the direction convention seems to be moving in? (see rust-lang/cargo#9282) I've changed it to my name for now.